testplan/web_ui/testing/src/AssertionPane/AssertionTypes/DictAssertions/FixCellRenderer.js

```diff
@@ -52,3 +82,116 @@ FixCellRenderer.propTypes = {
   /** The Column definition of the current cell */
   colDef: PropTypes.object,
 };
+
+
+/**
+ * Render a tooltip of a fix tag number.
+ */
+export const FixTagTooltip = (props) => {
+  if (_.isEmpty(props.names) && _.isEmpty(props.descr)) {
+    return null;
+  }
+
+  return (
+    <>
+      {props.names ? (
+        <span className={css(styles.tooltipTitle)}>
+          {props.names.join(' ')}
+        </span>
+      ) : null}
+      {props.descr ? (
+        <>
+          <br />
+          <span className={css(styles.tooltipDescription)}>
+            {props.descr}
+          </span>
+        </>
+      ): null}
+    </>
+  );
+};
+
+/**
+ * Render a tooltip of a fix tag value.
+ */
+export const FixTagValueTooltip = (props) => {
+  const [content, setContent] = useState(null);
+
+  if (content === null) {
+    axios
+      .get(`/api/v1/metadata/fix-spec/tags/${props.num}/enum-vals`)
+      .then((response) => {
+        const enum_vals = response.data;
+        let found = false;
+        for (let i = 0; i < enum_vals.length; i++) {
+          if (enum_vals[i].value === props.value.toString()) {
+            found = true;
+            setContent(
+              <span className={css(styles.tooltipDescription)}>
+                {enum_vals[i].descr || enum_vals[i].short_descr}
+              </span>
+            );
+            break;
```

Comment:
you can _return_ in here, thus you can remove the *found" variable。

Reply:
fixed